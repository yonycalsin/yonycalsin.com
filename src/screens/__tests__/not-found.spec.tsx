import { render } from '@testing-library/react'

import { TestProvider } from 'tests'
import NotFoundScreen from 'screens/not-found'

const setup = () => {
  return render(
    <TestProvider>
      <NotFoundScreen />
    </TestProvider>,
  )
}

describe('NotFoundScreen', () => {
  it('renders the not found screen', () => {
    const view = setup()

    expect(view.baseElement).toMatchSnapshot()
  })
})
