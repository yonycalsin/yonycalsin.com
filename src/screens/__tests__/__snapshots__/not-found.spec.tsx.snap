// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundScreen renders the not found screen 1`] = `
.emotion-0 {
  width: var(--yony-sizes-full);
  top: 0px;
  z-index: var(--yony-zIndices-banner);
  left: 0px;
  box-shadow: var(--yony-shadows-xs);
}

.chakra-ui-dark .emotion-0:not([data-theme]),
[data-theme=dark] .emotion-0:not([data-theme]),
.emotion-0[data-theme=dark] {
  box-shadow: var(--yony-shadows-lg);
}

.emotion-1 {
  width: 100%;
  -webkit-margin-start: auto;
  margin-inline-start: auto;
  -webkit-margin-end: auto;
  margin-inline-end: auto;
  max-width: var(--yony-sizes-container-md);
  -webkit-padding-start: var(--yony-space-4);
  padding-inline-start: var(--yony-space-4);
  -webkit-padding-end: var(--yony-space-4);
  padding-inline-end: var(--yony-space-4);
  padding-top: var(--yony-space-5);
  padding-bottom: var(--yony-space-5);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

@media screen and (min-width: 48em) {
  .emotion-1 {
    padding-top: var(--yony-space-8);
    padding-bottom: var(--yony-space-8);
  }
}

.emotion-2 {
  transition-property: var(--yony-transition-property-common);
  transition-duration: var(--yony-transition-duration-fast);
  transition-timing-function: var(--yony-transition-easing-ease-out);
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  outline: 2px solid transparent;
  outline-offset: 2px;
  color: inherit;
  display: contents;
}

.emotion-2:hover,
.emotion-2[data-hover] {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-2:focus-visible,
.emotion-2[data-focus-visible] {
  box-shadow: var(--yony-shadows-outline);
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.emotion-3>*:not(style)~*:not(style) {
  margin-top: 0px;
  -webkit-margin-end: 0px;
  margin-inline-end: 0px;
  margin-bottom: 0px;
  -webkit-margin-start: var(--yony-space-4);
  margin-inline-start: var(--yony-space-4);
}

.emotion-4 {
  transition-property: var(--yony-transition-property-common);
  transition-duration: var(--yony-transition-duration-fast);
  transition-timing-function: var(--yony-transition-easing-ease-out);
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  outline: 2px solid transparent;
  outline-offset: 2px;
  color: var(--yony-colors-primary-500);
  font-weight: var(--yony-fontWeights-bold);
}

.emotion-4:hover,
.emotion-4[data-hover] {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-4:focus-visible,
.emotion-4[data-focus-visible] {
  box-shadow: var(--yony-shadows-outline);
}

.emotion-6 {
  width: 100%;
  -webkit-margin-start: auto;
  margin-inline-start: auto;
  -webkit-margin-end: auto;
  margin-inline-end: auto;
  max-width: var(--yony-sizes-container-md);
  -webkit-padding-start: var(--yony-space-4);
  padding-inline-start: var(--yony-space-4);
  -webkit-padding-end: var(--yony-space-4);
  padding-inline-end: var(--yony-space-4);
  padding-top: var(--yony-space-12);
  padding-bottom: var(--yony-space-12);
}

.emotion-7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-7>*:not(style)~*:not(style) {
  margin-top: var(--yony-space-12);
  -webkit-margin-end: 0px;
  margin-inline-end: 0px;
  margin-bottom: 0px;
  -webkit-margin-start: 0px;
  margin-inline-start: 0px;
}

.emotion-8 {
  font-family: var(--yony-fonts-heading);
  font-weight: var(--yony-fontWeights-bold);
  font-size: var(--yony-fontSizes-3xl);
  line-height: 1.33;
}

@media screen and (min-width: 48em) {
  .emotion-8 {
    font-size: var(--yony-fontSizes-4xl);
    line-height: 1.2;
  }
}

.emotion-10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: var(--yony-space-6);
}

@media screen and (min-width: 48em) {
  .emotion-10 {
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
  }
}

.emotion-11 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-appearance: none;
  -moz-appearance: none;
  -ms-appearance: none;
  appearance: none;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  position: relative;
  white-space: nowrap;
  vertical-align: baseline;
  outline: 2px solid transparent;
  outline-offset: 2px;
  line-height: var(--yony-lineHeights-normal);
  border-radius: var(--yony-radii-md);
  font-weight: var(--yony-fontWeights-semibold);
  transition-property: var(--yony-transition-property-common);
  transition-duration: var(--yony-transition-duration-normal);
  height: auto;
  min-width: var(--yony-sizes-10);
  font-size: var(--yony-fontSizes-md);
  -webkit-padding-start: var(--yony-space-4);
  padding-inline-start: var(--yony-space-4);
  -webkit-padding-end: var(--yony-space-4);
  padding-inline-end: var(--yony-space-4);
  padding: 0px;
  color: var(--yony-colors-primary-200);
}

.emotion-11:focus-visible,
.emotion-11[data-focus-visible] {
  box-shadow: var(--yony-shadows-outline);
}

.emotion-11:disabled,
.emotion-11[disabled],
.emotion-11[aria-disabled=true],
.emotion-11[data-disabled] {
  opacity: 0.4;
  cursor: not-allowed;
  box-shadow: var(--yony-shadows-none);
}

.emotion-11:hover,
.emotion-11[data-hover] {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-11:hover:disabled,
.emotion-11[data-hover]:disabled,
.emotion-11:hover[disabled],
.emotion-11[data-hover][disabled],
.emotion-11:hover[aria-disabled=true],
.emotion-11[data-hover][aria-disabled=true],
.emotion-11:hover[data-disabled],
.emotion-11[data-hover][data-disabled] {
  background: initial;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-11:active,
.emotion-11[data-active] {
  color: var(--yony-colors-primary-500);
}

.emotion-12 {
  display: none;
}

@media screen and (min-width: 48em) {
  .emotion-12 {
    display: block;
  }
}

.emotion-16 {
  padding-top: var(--yony-space-12);
  padding-bottom: var(--yony-space-12);
}

.emotion-17 {
  width: 100%;
  -webkit-margin-start: auto;
  margin-inline-start: auto;
  -webkit-margin-end: auto;
  margin-inline-end: auto;
  max-width: var(--yony-sizes-container-md);
  -webkit-padding-start: var(--yony-space-4);
  padding-inline-start: var(--yony-space-4);
  -webkit-padding-end: var(--yony-space-4);
  padding-inline-end: var(--yony-space-4);
}

.emotion-18 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-18>*:not(style)~*:not(style) {
  margin-top: 0.5rem;
  -webkit-margin-end: 0px;
  margin-inline-end: 0px;
  margin-bottom: 0px;
  -webkit-margin-start: 0px;
  margin-inline-start: 0px;
}

.emotion-19 {
  font-weight: var(--yony-fontWeights-bold);
}

.emotion-20 {
  color: var(--yony-colors-gray-600);
}

.emotion-21 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-appearance: none;
  -moz-appearance: none;
  -ms-appearance: none;
  appearance: none;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  position: relative;
  white-space: nowrap;
  vertical-align: baseline;
  outline: 2px solid transparent;
  outline-offset: 2px;
  line-height: var(--yony-lineHeights-normal);
  border-radius: var(--yony-radii-md);
  font-weight: var(--yony-fontWeights-semibold);
  transition-property: var(--yony-transition-property-common);
  transition-duration: var(--yony-transition-duration-normal);
  height: auto;
  min-width: var(--yony-sizes-10);
  font-size: var(--yony-fontSizes-md);
  -webkit-padding-start: var(--yony-space-4);
  padding-inline-start: var(--yony-space-4);
  -webkit-padding-end: var(--yony-space-4);
  padding-inline-end: var(--yony-space-4);
  padding: 0px;
  color: var(--yony-colors-green-200);
}

.emotion-21:focus-visible,
.emotion-21[data-focus-visible] {
  box-shadow: var(--yony-shadows-outline);
}

.emotion-21:disabled,
.emotion-21[disabled],
.emotion-21[aria-disabled=true],
.emotion-21[data-disabled] {
  opacity: 0.4;
  cursor: not-allowed;
  box-shadow: var(--yony-shadows-none);
}

.emotion-21:hover,
.emotion-21[data-hover] {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-21:hover:disabled,
.emotion-21[data-hover]:disabled,
.emotion-21:hover[disabled],
.emotion-21[data-hover][disabled],
.emotion-21:hover[aria-disabled=true],
.emotion-21[data-hover][aria-disabled=true],
.emotion-21:hover[data-disabled],
.emotion-21[data-hover][data-disabled] {
  background: initial;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-21:active,
.emotion-21[data-active] {
  color: var(--yony-colors-green-500);
}

.emotion-22 {
  font-family: var(--yony-fonts-mono);
  background: whiteAlpha;
  border-radius: var(--yony-radii-md);
  border-width: 1px;
  border-bottom-width: 3px;
  font-size: 0.8em;
  font-weight: var(--yony-fontWeights-bold);
  line-height: var(--yony-lineHeights-normal);
  -webkit-padding-start: 0.4em;
  padding-inline-start: 0.4em;
  -webkit-padding-end: 0.4em;
  padding-inline-end: 0.4em;
  white-space: nowrap;
  -webkit-margin-start: var(--yony-space-2);
  margin-inline-start: var(--yony-space-2);
  -webkit-margin-end: var(--yony-space-2);
  margin-inline-end: var(--yony-space-2);
}

.emotion-24 {
  display: none;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  position: fixed;
  bottom: var(--yony-space-8);
  left: var(--yony-space-8);
}

.emotion-24>*:not(style)~*:not(style) {
  margin-top: var(--yony-space-5);
  -webkit-margin-end: 0px;
  margin-inline-end: 0px;
  margin-bottom: 0px;
  -webkit-margin-start: 0px;
  margin-inline-start: 0px;
}

@media screen and (min-width: 62em) {
  .emotion-24 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
  }
}

.emotion-25 {
  transition-property: var(--yony-transition-property-common);
  transition-duration: var(--yony-transition-duration-fast);
  transition-timing-function: var(--yony-transition-easing-ease-out);
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  outline: 2px solid transparent;
  outline-offset: 2px;
  color: inherit;
}

.emotion-25:hover,
.emotion-25[data-hover] {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-25:focus-visible,
.emotion-25[data-focus-visible] {
  box-shadow: var(--yony-shadows-outline);
}

<body
  class="chakra-ui-dark"
>
  <div>
    <div
      class="emotion-0"
    >
      <div
        class="chakra-container emotion-1"
      >
        <a
          class="chakra-link emotion-2"
          href="/"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2732%27%20height=%2732%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt=""
              data-nimg="intrinsic"
              decoding="async"
              src="/_next/image?url=%2Fimg.jpg&w=64&q=75"
              srcset="/_next/image?url=%2Fimg.jpg&w=32&q=75 1x, /_next/image?url=%2Fimg.jpg&w=64&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </span>
        </a>
        <div
          class="chakra-stack emotion-3"
        >
          <a
            class="chakra-link emotion-4"
            href="/projects"
          >
            Projects
          </a>
          <a
            class="chakra-link emotion-4"
            href="/me"
          >
            About Me
          </a>
        </div>
      </div>
    </div>
    <div>
      <main
        class="chakra-container emotion-6"
      >
        <div
          class="chakra-stack emotion-7"
        >
          <h2
            class="chakra-heading emotion-8"
          >
            😢
          </h2>
          <h2
            class="chakra-heading emotion-8"
          >
            Page not found
          </h2>
          <div
            class="emotion-10"
          >
            <a
              class="chakra-button emotion-11"
              href="/"
            >
              Go to home
            </a>
            <p
              class="chakra-text emotion-12"
            >
              •
            </p>
            <a
              class="chakra-button emotion-11"
              href="/me"
            >
              Go to about-me page
            </a>
            <p
              class="chakra-text emotion-12"
            >
              •
            </p>
            <a
              class="chakra-button emotion-11"
              href="/projects"
            >
              Go to projects page
            </a>
          </div>
        </div>
      </main>
    </div>
    <footer
      class="emotion-16"
    >
      <div
        class="chakra-container emotion-17"
      >
        <div
          class="chakra-stack emotion-18"
        >
          <p
            class="chakra-text emotion-19"
          >
            Created By Yony Calsin
          </p>
          <p
            class="chakra-text emotion-20"
          >
            Built with Next.js, MDX, Chakra-Ui and Vercel
          </p>
          <button
            class="chakra-button emotion-21"
            type="button"
          >
            Press 
            <kbd
              class="chakra-kbd emotion-22"
            >
              Ctrl
            </kbd>
            +
            <kbd
              class="chakra-kbd emotion-22"
            >
              K
            </kbd>
             to start →
          </button>
        </div>
      </div>
    </footer>
    <div
      class="chakra-stack emotion-24"
    >
      <a
        class="chakra-link emotion-25"
        href="https://github.com/yonycalsin"
        rel="noreferrer"
        target="_blank"
      >
        Github
      </a>
      <a
        class="chakra-link emotion-25"
        href="https://www.linkedin.com/in/yonycalsin"
        rel="noreferrer"
        target="_blank"
      >
        Linkedin
      </a>
      <a
        class="chakra-link emotion-25"
        href="https://twitter.com/yonycalsin"
        rel="noreferrer"
        target="_blank"
      >
        Twitter
      </a>
      <a
        class="chakra-link emotion-25"
        href="mailto:helloyonycalsin@gmail.com"
        rel="noreferrer"
        target="_blank"
      >
        Email
      </a>
    </div>
    <span
      hidden=""
      id="__chakra_env"
    />
  </div>
  <div
    class="chakra-portal"
  >
    <ul
      aria-live="polite"
      id="chakra-toast-manager-top"
      role="region"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
    />
    <ul
      aria-live="polite"
      id="chakra-toast-manager-top-left"
      role="region"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
    />
    <ul
      aria-live="polite"
      id="chakra-toast-manager-top-right"
      role="region"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
    />
    <ul
      aria-live="polite"
      id="chakra-toast-manager-bottom-left"
      role="region"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
    />
    <ul
      aria-live="polite"
      id="chakra-toast-manager-bottom"
      role="region"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
    />
    <ul
      aria-live="polite"
      id="chakra-toast-manager-bottom-right"
      role="region"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
    />
  </div>
</body>
`;
